pkg/3scale/amp/component/images.go

```diff
@@ -12,6 +12,10 @@ func SystemImageURL() string {
 	return "quay.io/3scale/porta:latest"
 }
 
+func SystemSearchdImageURL() string {
+	return "quay.io/3scale/searchd:nightly"
```

Comment:
In this case `nightly` is better because it is generated periodically, thus catching any potential porta changes. Also trying to follow best practice to avoid `latest` that can be confusing. Is it latest stable or unstable, etc.

Reply:
we had this discussion earlier and we agreed on each component having `latest` of the component they own. `latest` has specific pull policy so we make sure it is always fetched from the registry (we could achieve the same for `nightly` with some custom config)

I do not have a strong opinion on this, but I would use `latest` like any other 3scale component.